pkg/process/metadata/workloadmeta/grpc.go

```diff
@@ -0,0 +1,21 @@
+package workloadmeta
```

Comment:
```suggestion
// Unless explicitly stated otherwise all files in this repository are licensed
// under the Apache License Version 2.0.
// This product includes software developed at Datadog (https://www.datadoghq.com/).
// Copyright 2016-present Datadog, Inc.

package workloadmeta
```

Reply:
Thank you, I tend to miss these a lot.